build.sbt

```diff
@@ -21,10 +21,9 @@ libraryDependencies ++= Seq(
   "org.scalatest"         %% "scalatest"  % "3.0.1" % "test",
   "org.scalacheck"        %% "scalacheck" % "1.13.4" % "test",
   "com.typesafe"          % "config"      % "1.3.1",
-  "com.github.scopt"      %% "scopt"      % "3.7.0",
   "com.github.pureconfig" %% "pureconfig" % "0.8.0",
-  "com.beachape"          %% "enumeratum" % "1.5.12",
-  "org.typelevel"         %% "cats-core"  % "1.0.0-MF"
```

Comment:
Lets be explicit about the cats version and leave this in.

Reply:
The explicit cats version here conflicts with pureconfig, which uses 0.9.0, and breaks the app.
The cats stuff we are doing still works in 0.9.0, so I could:
```"org.typelevel"         %% "cats-core"  % "0.9.0"``` ?